CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added an OnChange event handler for the TextArea React Kit ([#505][] @jasperfurniss)
```

Comment:
👍

Reply:
Nice! Please create a `### Changed` category under `[Unreleased]` section and move it there?


```
### Changed 

- Added an OnChange event handler for the TextArea React Kit ([#505][] @jasperfurniss)

[#505]: https://github.com/powerhome/playbook/pull/505